docs/getting-started/configuration.md

```diff
@@ -0,0 +1,307 @@
+ id: 'github-auth-provider',
+ title: 'GitHub',
+ message: 'Sign in using GitHub',
+ apiRef: githubAuthApiRef,
+};
+```
+
+Search for `const app = createApp({` in this file, and below `apis,` add:
+
+```
+components: {
+   SignInPage: props => (
+     <SignInPage
+       {...props}
+       auto
+       provider={githubProvider}
+     />
+   ),
+ },
+```
+
+That should be it. You can stop your Backstage App. When you start it again and
+go to your Backstage portal in your browser, you should have your login prompt!
+
+To learn more about Authentication in Backstage, there's the following docs you
+could read:
+
+- [Adding Authentication](../auth/)
+- [Adding a new Authentication Provider](../auth/add-auth-provider)
+- [Using authentication and identity](../auth/using-auth)
+- [Using organizational data from GitHub](../integrations/github/org.md)
+
+### Setting up a GitHub Integration
+
+The GitHub integration supports loading catalog entities from GitHub or GitHub
+Enterprise. Entities can be added to static catalog configuration, registered
+with the catalog-import plugin, or discovered from a GitHub organization. Users
+and Groups can also be loaded from an organization. While using GitHub Apps
+might be the best way to set up integrations, for this tutorial you'll use a
+Personal Access Token.
+
+Create your Personal Access Token by opening the
+[the GitHub token creation page](https://github.com/settings/tokens/new). Use a
+name to identify this token and put it in the notes field. Choose a number of
+days for expiration. If you have a hard time picking a number, we suggest to go
+for 7 days, it's a lucky number.
+
+<p align='center'>
+  <img src='../assets/getting-started/gh-oauth.png' alt='Screenshot of the GitHub OAuth creation page'>
+</p>
+
+Set the scope to your likings. For this tutorial, selecting "repo" should be
+enough.
+
+In the `app-config.yaml`, search for `integrations:` and add your token, like we
+did in below example:
+
+```
+integrations:
+  github:
+    - host: github.com
+      token: ghp_urtokendeinfewinfiwebfweb
+```
+
+That's settled. This information will be leveraged by other plugins.
+
+Some helpful links, for if you want to learn more about:
+
+- [Other available integrations](../integrations/)
+- [Using GitHub Apps instead of a Personal Access Token](../plugins/github-apps#docsNav)
+
+### Explore what we've done so far
+
+Open your Backstage frontend. You should see your login screen if you're not
+logged in yet. As soon as you've logged in, go to Settings, you'll see your
+profile. Hopefully You'll recognize the profile picture and name on your screen,
+otherwise something went terribly wrong.
+
+Register an existing component
+
+- Register a new component, by going to `create` and choose
+  `Register existing component`
+
+<p align='center'>
+  <img src='../assets/getting-started/b-existing-1.png' alt='Software template main screen, with a blue button to add an existing component'>
```

Comment:
Protip, if you want images to be zoomable on click. Example https://backstage.io/docs/features/techdocs/architecture 😁 

```suggestion
  <img data-zoomable src='../assets/getting-started/b-existing-1.png' alt='Software template main screen, with a blue button to add an existing component'>
```

Reply:
```suggestion
  <img src='../assets/getting-started/b-existing-1.png' alt='Software template main screen, with a blue button to add an existing component' />
```